R/utilities.R

```diff
@@ -4418,25 +4371,29 @@ classify_fl <- function(
 #' @param projection The projection of the samples. Only used to retrerive data through GAMBLR when it is not provided. Defaults to grch37.
 #' @param output The output to be returned after prediction is done. Can be one of predictoins, matrix, or both. Defaults to predictions.
 #' @param adjust_ploidy Whether to perform ploidy adjustment for the CNV data. Defaults to TRUE (recommended).
+#' @param include_N1 When running Lacy classifieer, indicate whether to set samples with NOTCH1 mutations to N1 group as described in Runge et al (2021). Defaults to FALSE.
```

Comment:
Clarify that it's NOTCH1 truncating (I guess frameshift and nonsense?)

Reply:
This is what I was trying to figure out yesterday too. Lacy/Runge considered only frameshift, and this is what I implemented here too. The truncating status is now clarified and I will push this update